README.md

```diff
@@ -363,6 +363,35 @@ Fields that are empty will have a value sent to the writer of an empty
 array (`[]`). Writers that need to special-case empty fields should do so in the
 writer class in question.
 
+### Error Handling
+
+During the mapping phase, the default behavior when an otherwise-unhandled ("unexpected") exception occurs is to log some information about the error and the record that was being mapped, then terminate processing.
+
+This situation usually indicates a bug in the code that handles MARC-Sorl transformations that you should address, but there are situations where this behavior may not fit your needs.
+
+You have the option (after carefully considering the implications!) of adding
+the `mapping_error_handler` setting to your `Indexer`, which should be a
+lambda that accepts a `Context`, the current index step, and the thrown
+exception, e.g.
+
+``` 
+# log errors until we exceed some value
+max_error_count = 5
+errors = 0
+  indexer.settings['mapping_error_handler] = lambda do |ctx, step, e|
+    errors += 1
+      ctx.logger.warn("I got #{e} during #{step.inspect} but that is not going to bother me one bit")
+      if errors > max_error_count
```

Comment:
believe this is supposed to be `<` than `>`?

Reply:
Yeah, probably, if you're into the whole "do what the variable name suggests" thing =)

Reply:
The signature of the lambda matches what was in scope for the original `log_mapping_errors` method, with the exception tacked onto the end: https://github.com/traject/traject/blob/master/lib/traject/indexer.rb#L365

Reply:
It's possible the original didn't really need that second param either. Note that the code that originally calls `log_mapping_errors` sets `context.index_step` to the same thing as second param, right before it calls it and passes context in too. 

https://github.com/traject/traject/blob/d7d010747a9fde23db77a2e7eaa7e321d0059405/lib/traject/indexer.rb#L344-L345